docs/onecodex.lib.rst

```diff
@@ -0,0 +1,53 @@
+onecodex.lib package
```

Comment:
I might be misunderstanding the purpose of `onecodex.lib`, but it seems like more of an internal API vs. something we'd want to document and expose to users.

Are there other subpackages we *don't* want to document because they're internal?